testing/src/cypress/custom-command/common/authorization.ts

```diff
@@ -0,0 +1,27 @@
+import axios, { AxiosResponse } from "axios";
+import { stringify } from "querystring";
+import authData from "../../e2e/data/authentication-details.json";
```

Comment:
The JSON data should be declared either camel-case or pascal case. I saw both ways along the code.

Reply:
updated